miden-lib/asm/miden/kernels/tx/tx.masm

```diff
@@ -257,6 +261,10 @@ end
 #! - if note_tag starts with anything but 0b11 and note_type is not public.
 #! - the number of created notes exceeds the maximum limit of 4096.
 export.create_note
+    # TODO: we execute `push.0 drop` before `emit` as decorators are not supported without other
+    # instructions - see: https://github.com/0xPolygonMiden/miden-vm/issues/1122
+    push.0 drop emit.BEFORE_NEW_NOTE_EVENT
```

Comment:
Here and in other similar places, let's use the convention from [here](https://github.com/0xPolygonMiden/miden-base/blob/546f3195684ad3d52b2e710cc730f5b03184112a/miden-lib/asm/kernels/transaction/main.masm#L65) for explaining why the extra operations are needed.